docs/stream-aggregation.md

```diff
@@ -418,11 +418,13 @@ For example:
 <img alt="total aggregation counter reset" src="stream-aggregation-check-total-reset.png">
 
 The same behavior will occur when creating or deleting new series in an aggregation group -
-`total` will increase monotonically considering the values of the series set. 
-
+`total` will increase monotonically considering the values of the series set.  
 An example of changing a set of series can be restarting a pod in the Kubernetes.
 This changes a label with pod's name in the series, but `total` account for such a scenario and do not reset the state of aggregated metric.
 
+Aggregating irregular and sporadic metrics (received from [Lambdas](https://aws.amazon.com/lambda/)
+or [Cloud Functions](https://cloud.google.com/functions)) can be controlled via `staleness_inteval`.
```

Comment:
I think there should be a link here to `#stream-aggregation-config`

Reply:
updated